kubernetes_deploy/production/deployment.yml

```diff
@@ -87,3 +87,8 @@ spec:
             secretKeyRef:
               name: s3
               key: region
+        - name: AMQP_BROKER_URL
```

Comment:
Do we need this in development too? :trollface:

Reply:
Yes
It's already there
:trollface:

Reply:
```
- name: AMQP_BROKER_URL
   value: "amqp://guest:guest@laa-legal-adviser-api-shared-services:5672//"
```